effortless-chef-io/effortless-config.md

```diff
@@ -125,7 +141,7 @@ This pattern is used to build policyfiles using the chef_repo pattern for organi
    echo "Installing $latest_hart_file"
    hab pkg install $latest_hart_file
 
-   echo "Determing pkg_prefix for $latest_hart_file"
+   echo "Determining pkg_prefix for $latest_hart_file"
```

Comment:
@TheLunaticScripter Can we check it this typo is in the code?

Reply:
yep I'll take a look.